server/index.js

```diff
@@ -1,10 +1,7 @@
 // Require all the packages
 const http = require('http');
+const path = require('path');
 const express = require('express');
-const OBA = require('oba-api');
-const debug = require('debug')('robat');
```

Comment:
To fix the linting errors ^^

Also, the OBA api handler is probably going to be used in a separate file and there wasn't a single debug statement necessary in the main index file at this point